src/molecules/actions-menu/styles.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+export const ActionsMenuStyle = styled.div`
+  display: inline-block !important;
```

Comment:
@henriquemacedo any reason why we need the important here ?

Reply:
For now, we can remove it. But we need to make attention when we use it somewhere because the parent can influence his behavior.